lib/castle/commands/review.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module Castle
+  module Commands
+    class Review
+      def initialize(_context)
+      end
+
+      def build(options = {})
```

Comment:
why add options instead of just passing the `review_id`?

Reply:
We should be consistent with our interface. It should be the same for every endpoint.

Reply:
well the other ones are all post endpoints and this is a get. I would argue that they are semantically different. `Stripe::Charge.retrieve("ch_18atAXCdGbJFKhCuBAa4532Z")` this is how Stripe does it. I think it is more self-explanatory

Reply:
Fair enough.